src/tools/ScenarioMeasurement/Startup/Startup.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
+    <LangVersion>Preview</LangVersion>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Diagnostics.Tracing.TraceEvent" Version="2.0.41" />
```

Comment:
nit: I think you do not need this here because it's already included in the `Utils.csproj`

Reply:
Good catch. I chose to remove it from `Utils.csproj` instead as that project has no usages and could be used in non-TraceEvent based things. (If I refactor more in the future I will maybe make a `TracingUtils` library.)